toonz/sources/toonz/exportlevelpopup.h

```diff
@@ -86,52 +94,55 @@ class ExportLevelPopup final : public FileBrowserPopup {
   Swatch *m_swatch;
 
   // Others
-
+  std::vector<TXshSimpleLevel *> outputLevels;
+  bool allPlis;
   std::map<std::string, TPropertyGroup *> m_formatProperties;
 
   TFrameHandle m_levelFrameIndexHandle;  //!< Autonomous current level's frame
                                          //!\a index handle.
 
 private slots:
 
+  virtual void updateOnSelection();
   void onOptionsClicked();
   void onRetas(int);
   void initFolder() override;
-  void updateOnSelection();
   void onformatChanged(const QString &);
   void checkAlpha();
   void updatePreview();
 };
 
-//-----------------------------------------------------------------------------
+//********************************************************************************
+//    ExportOptions  definition
+//********************************************************************************
 
 class ExportLevelPopup::ExportOptions final : public QFrame {
   Q_OBJECT
 
 public:
   ExportOptions(QWidget *parent = 0);
-
+  bool pliOtionsVisable;  // be used when updateonselection,ExportOptions show,
```

Comment:
```suggestion
  bool pliOptionsVisible; // indicates visibility of PLI (vector) export options based on current selection
```

Would also need to be updated elsewhere.